app/src/ui/history/commit-summary.tsx

```diff
@@ -304,12 +304,18 @@ export class CommitSummary extends React.Component<
     return (
       <div id="commit-summary" className={className}>
         <div className="commit-summary-header">
-          <RichText
```

Comment:
Same commit as above.. other thoughts could make a commit.friendlySummary if you have the whole `commit` class at these points.